app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt

```diff
@@ -93,6 +93,10 @@ class MapsFragment : androidx.fragment.app.Fragment(), OnClickListener, OnMarker
     override fun onPause() {
         super.onPause()
 
+        if(this.map != null){
+            this.map?.isMyLocationEnabled = false
```

Comment:
Is onMapReady get called again when the app resumes? If not, the location updates might not trigger again.

Reply:
Yes, getMapAsync in onResume takes care of that (onMapReady is the async callback)